code/modules/vr/vr_headset.dm

```diff
@@ -0,0 +1,89 @@
+//Glorified teleporter that puts you in a new human body.
+// it's """VR"""
+
+/obj/item/clothing/ears/vr_headset
+	desc = "Your ticket to another reality. Designed to be worn above the ears."
+	name = "VR Headset"
+	icon_state = "brainset"
+	item_state = "brainset"
+	actions_types = list(/datum/action/item_action/enter_vr)
+	var/you_die_in_the_game_you_die_for_real = FALSE //Perhaps some day
```

Comment:
I like this var name

Reply:
At this time I don't have an implementation that is not broken, so it lies there... waiting